Changes.md

```diff
@@ -8,6 +8,7 @@ Features
   - Double clicking a node name will open a Node Editor popup.
   - Dragging a node name into the Graph Editor will zoom to the node.
   - Double clicking, pressing <kbd>Return</kbd> or <kbd>Enter</kbd> on a parameter value or operation will open a plug popup to edit the value.
+- Edit Menu : Added command to duplicate node(s) with each new node plug connected to the original node plug input, with accompanying keyboard shortcut.
```

Comment:
Perhaps "with accompanying keyboard shortcut" could just be "with <kbd>Ctrl</kbd>+<kbd>D</kbd> shortcut" since it's shorter but more informative?

Reply:
Fixed in 60dc668427f0fe3265e595d415d647521380ddfa